.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,25 @@
+# this setting will force users to use the provided issue templates
+blank_issues_enabled: true
+# This setting may be changed in the future if the templates are not being used!
```

Comment:
BEWARE: I'll be changing this to false if users improperly bypass the form templates with our blank issue option.

As maintainers, we could still retain the ability to open a blank issue template if this is set to false. (see comments at EoF)